app/src/main/java/com/ds/avare/webinfc/WebAppMapInterface.java

```diff
@@ -163,7 +163,7 @@ public void handleMessage(Message msg) {
 
                 mWebView.loadUrl("javascript:plan_clear()");
                 String func = "javascript:setData('" +
-                        Helper.formatJsArgs(data.airport) + "','" +
+                        Helper.formatJsArgs(data.destinationName) + "','" +
```

Comment:
treat VOR name just like any other airport name displayed at the top of Long-pressed HTML page